django/db/backends/dummy/base.py

```diff
@@ -83,4 +83,4 @@ def __init__(self, *args, **kwargs):
         self.validation = BaseDatabaseValidation(self)
 
     def is_usable(self):
-        return True
+        return False
```

Comment:
Not sure if this might mask some exceptions, but here's what I came up with for now:
```python
for db in connections:
    connection = connections[db]
    try:
        connection.cursor()
    except ImproperlyConfigured:
        # Raised by the dummy backend.
        pass
    else:
        loader.check_consistent_history(connection)
```